src/app/tests/suites/certification/Test_TC_CDOCONC_1_1.yaml

```diff
@@ -0,0 +1,202 @@
+
+    - label:
+          "Given CDOCONC.S.F03(CRI) ensure featuremap has the correct bit set"
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      PICS: CDOCONC.S.Afffc && CDOCONC.S.F03
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x8]
+
+    - label:
+          "Given CDOCONC.S.F04(PEA) ensure featuremap has the correct bit set"
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      PICS: CDOCONC.S.Afffc && CDOCONC.S.F04
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x10]
+
+    - label:
+          "Given CDOCONC.S.F05(AVG) ensure featuremap has the correct bit set"
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      PICS: CDOCONC.S.Afffc && CDOCONC.S.F05
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x20]
+
+    - label: "Read the global attribute: AttributeList"
+      command: "readAttribute"
+      attribute: "AttributeList"
+      PICS: CDOCONC.S.Afffb
+      response:
+          constraints:
+              type: list
+              contains: [9, 65528, 65529, 65530, 65531, 65532, 65533]
+
+    - label: "Read the optional attribute Uncertainty in AttributeList"
+      command: "readAttribute"
+      attribute: "AttributeList"
+      PICS: CDOCONC.S.Afffb && CDOCONC.S.A0007 && CDOCONC.S.F00
+      response:
+          constraints:
+              type: list
+              contains: [7]
+
+    - label:
+          "Read the optional, feature dependent attributes MeasuredValue,
+          MinMeasuredValue, MaxMeasuredValue and Measurement Unit in
+          AttributeList"
+      command: "readAttribute"
+      attribute: "AttributeList"
+      PICS: CDOCONC.S.Afffb && CDOCONC.S.F00
+      response:
+          constraints:
+              type: list
+              contains: [0, 1, 2, 8]
+
+    - label:
+          "Read the optional, feature dependent attributes PeakMeasuredValue &
+          PeakMeasuredValueWindow in AttributeList"
+      command: "readAttribute"
+      attribute: "AttributeList"
+      PICS: CDOCONC.S.Afffb && CDOCONC.S.F04
+      response:
+          constraints:
+              type: list
+              contains: [3, 4]
+
+    - label:
+          "Read the optional, feature dependent attributes AverageMeasuredValue
+          AverageMeasuredValueWindow in AttributeList"
+      command: "readAttribute"
+      attribute: "AttributeList"
+      PICS: CDOCONC.S.Afffb && CDOCONC.S.F05
+      response:
+          constraints:
+              type: list
+              contains: [5, 6]
+
+    - label:
+          "Read the optional, feature dependent attribute LevelValue in
+          AttributeList"
+      command: "readAttribute"
+      attribute: "AttributeList"
+      PICS: CDOCONC.S.Afffb && CDOCONC.S.F01
+      response:
+          constraints:
+              type: list
+              contains: [10]
```

Comment:
Should test this is absent if CDOCONC.S.F01 is not set.

Reply:
I have added an excludes test for when CDOCONC.S.F01 is not set